setup.cfg

```
@@ -42,7 +42,7 @@ install_requires =
     notebook_shim>=0.1
     jinja2>=3.0.3
     ypy-websocket
-    jupyter_ydoc==0.1.10
+    jupyter_ydoc==0.1.12
```

Did you mean v0.1.13?
```suggestion
    jupyter_ydoc==0.1.13
```

---

yep

---

Another option would be `jupyter_ydoc>=0.1.13,<0.2.0`, so that we don't update so often in JupyterLab.
And we make sure that each time we change the document structures in jupyter-ydoc, we do a minor bump.